utils/clean_up_repo.py

```diff
@@ -0,0 +1,32 @@
+"""
+The Qxf2 automation repository ships with example tests.
+Run this file to delete all the example files and start fresh with your example.
+After this script runs, you will need to edit a few files to configure them to suit your own repository
+Usage: python clean_up_repo.py
+"""
+import os,sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from conf import clean_up_repo_conf as conf
+
+def delete_file(filename):
+    # The method will delete a particular file
+    if os.path.exists(filename):
+        os.remove(filename)
+        print(f'{filename} deleted')
+
+def delete_files_in_dir(directory, files):
+    # The method will delete files in a particular directory
+    for filename in files:
+        delete_file(os.path.join(directory,filename))
+
+def delete_files_used_in_example():
+    # The method will delete files mentioned in the blog
+    # Blog URL: https://qxf2.com/blog/how-to-start-using-the-qxf2-framework-with-a-new-project/
```

Comment:
Why is this new style of docstring being adopted? I thought we agreed to leave the multi-line comments as is. We use double quotes everywhere else in the repository. So lets just use that.

Reply:
@qxf2 , I guess we agreed to leave multiline comments as it is for the first line only. The style of the docstring is being already used in our framework: e.g.`copy_framework_template.py` 

```
"""
This script would copy the required framework files from the input source to the input destination given by the user.
1. Copy files from POM to the newly created destination directory.
2. Verify if the destination directory is created and create the sub-folder to copy files from POM\Conf.
3. Verify if the destination directory is created and create the sub-folder to copy files from POM\Page_Objects.
4. Verify if the destination directory is created and create the sub-folder to copy files from POM\Utils.

"""
import os,sys
import shutil
from optparse import OptionParser
sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
from conf import copy_framework_template_conf as conf

def copy_framework_template(src_folder,dst_folder):
	"run the test"
	#1. Copy files from POM to the newly created destination directory.
	#1a. Get details from conf file
	src_files_list = conf.src_files_list
		
	#1b. Create the new destination directory
	os.makedirs(dst_folder)
```
Hence I have to follow it in this file as well.

Reply:
@qxf2 , let me know in case you have more comments around this.